deploy-board/deploy_board/templates/landing.html

```diff
@@ -18,19 +18,29 @@
     </div>
 </div>
 
-    <div class="panel panel-default">
-    {% include "panel_heading.tmpl" with panel_title="Deployment Statistics" panel_body_id="deployStatsId" direction="down" %}
-        <div id="deployStatsId" class="collapse in panel-body">
-            <div id="dailyDeployCountId"></div>
-        </div>
+<div class="panel panel-default">
```

Comment:
can we also show count of ONGOING sidecars and all deploys?

Reply:
there is limit number of sidecar so the the ongoing sidecar count should be very small so the table should be sufficient.  since it is the home page, also want to limit the DB/api call to ensure the fast load of the home page.

Reply:
Sure, It probably does not matter much for sidecar today but for ongoing deploy number in general for all services might be helpful. Not sure if we need additional db query for that, since we already have a list of ongoing deploys retrieved.

Reply:
nm, I see ongoing deploys is a paginated query for good reason. So, yeah, I think we can avoid separate query for pulling count for now.